src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -0,0 +1,418 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Test.Utilities;
+using Xunit;
+using System.Collections.Generic;
+using System;
+using System.Collections.Immutable;
+using System.Globalization;
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public class SymbolEqualityTests : CSharpTestBase
```

Comment:
Please add some tests for getting symbols from metadata, as opposed to just symbols from source.

Reply:
Also please test the rest of the public symbol types. Properties, locals, parameters, anything with a TypeSymbol nested in it.

---
In reply to: [305126999](https://github.com/dotnet/roslyn/pull/37247#discussion_r305126999) [](ancestors = 305126999)

Reply:
Already have _from_metadata tests. We havn't changed Properties, locals etc. yet, so they won't work. We should probably create a bug to track that. 

---
In reply to: [305126999](https://github.com/dotnet/roslyn/pull/37247#discussion_r305126999) [](ancestors = 305126999)